.eslintrc.js

```diff
@@ -1,48 +1,8 @@
 module.exports = {
-  root: true,
```

Comment:
I highly recommend against this. These are the Ember defaults, and most apps and addons adhere to them. If there are specific rules we need to add on top, we could do so or maybe even use the DSC plugin alongside these.

Reply:
I will follow up with my team and add that to `eslint-config-dollarshaveclub` if it makes sense for DSC—per your recommendation.

Reply:
I'm not sure if it makes sense for DSC, but it does make sense for this and your other popular, open source, Ember addons. They are used heavily by the community and have several active contributors. I think it would be preferential to keep the linting the same it has been, in line with the Ember community, here.